ash/src/extensions/khr/draw_indirect_count.rs

```diff
@@ -0,0 +1,73 @@
+#![allow(dead_code)]
+use crate::version::{DeviceV1_0, InstanceV1_0};
+use crate::vk;
+use std::ffi::CStr;
+use std::mem;
+
+
+#[derive(Clone)]
+pub struct DrawIndirectCount {
+    handle: vk::Device,
+    draw_indirect_count_fn: vk::KhrDrawIndirectCountFn,
+}
+
+impl DrawIndirectCount {
+
+    pub fn new<I: InstanceV1_0, D: DeviceV1_0>(instance: &I, device: &D) -> DrawIndirectCount {
+        let draw_indirect_count_fn = vk::KhrDrawIndirectCountFn::load(|name| unsafe {
+            mem::transmute(instance.get_device_proc_addr(device.handle(), name.as_ptr()))
+        });
+
+        DrawIndirectCount {
+            handle: device.handle(),
+            draw_indirect_count_fn,
+        }
+    }
+
+    pub fn name() -> &'static CStr {
+        vk::KhrDrawIndirectCountFn::name()
+    }
+
+    pub unsafe fn cmd_draw_indexed_indirect_count(
```

Comment:
Can you links to the _individual_ man pages? Here is an example:

https://github.com/MaikKlein/ash/blob/31d14486e5c03c64d6bed9aa59d2c473abcd4339/ash/src/extensions/khr/swapchain.rs#L69-L70